20230726/s2-1690376653994/README.md

```diff
@@ -0,0 +1,57 @@
+# Scenario 2 - ALS Baseline with Sims-only
```

Comment:
I think it will be more meaningful if we change it to "ALS Baseline with Sims". InScenario 1 as it was only Sims without ALS and in this scenario the baseline is with ALS.

Reply:
Noted.